### PR TITLE
server: reuse code for container removal

### PR DESCRIPTION
**Pull request by haircommander**
#### What type of PR is this?

/kind cleanup
#### What this PR does / why we need it:
more details in each commit
#### Does this PR introduce a user-facing change?

```release-note
none
```

### CODE REVIEW

File: internal/lib/remove.go [hunk 1]
```diff
@@ -9,45 +9,28 @@ import (
...
-		if force {
-			if err = c.StopContainer(ctx, ctr, 10); err != nil {
-				return "", errors.Wrapf(err, "unable to stop container %s", ctrID)
-			}
-		} else {
-			return "", errors.Errorf("cannot remove running container %s", ctrID)
-		}
-	}
-
 	if err := c.runtime.DeleteContainer(ctx, ctr); err != nil {
```
Comment (TomSweeneyRedHat) [inline internal/lib/remove.go:17]: Are these tests still done elsewhere now?

Comment (haircommander) [reply internal/lib/remove.go:17]: yeah they're done inside of the runtime with  calls to "ShouldBeStopped"

**Comment by saschagrunert** [comment]
A friendly reminder that this PR had no activity for 30 days.

**Comment by haircommander** [comment]
closing in favor of https://github.com/cri-o/cri-o/pull/5740

**Pull request closed without merging**